### DISCUSSION TITLE
Have renovate update only the dashboard issue.

### DISCUSSION THREAD

**Discussion by juchom** [renovatebot/renovate#30640] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Hello,

I would like to have renovate only update the dependency issue and let me select the PR I want to create from this issue.

Thanks in advance

**Comment by rarkins** [comment, marked as answer]
dependencyDashboardApproval=true

**Reply by juchom** [reply]
Thanks for the quick reply.

I'm sorry, I was looking for this option and did not find it.

Thanks a lot

**Reply by HonkingGoose** [reply]
Read the [Renovate docs, key concepts, Dependency Dashboard](https://docs.renovatebot.com/key-concepts/dashboard/) to learn more.
